src/Avatar.svelte

```diff
@@ -1,14 +1,19 @@
 <script>
   import i from 'initials';
+  import {getRandomColor} from './utils.js';
 
   export let style = '';
   export let name = 'Avatar';
   export let initials;
   export let src;
-  export let bgColor = 'lightGray';
+  export let bgColor;
   export let textColor = 'white';
   export let size = '50px';
   export let borderRadius = '50%';
+  export let square = false;
+  export let colored = false;
```

Comment:
Maybe we can use a better name to this property, for indicate for the user that this property is related to a random color for background (eg. randomBgColor or something like that)

Reply:
okay, I'll revise it 👌